src/dialogs/ADialog.ts

```diff
@@ -26,13 +26,25 @@ abstract class ADialog {
     });
   }
 
-  protected static registerPopup(popup: HTMLElement, replace: boolean) {
+  protected static registerPopup(popup: HTMLElement, popper: Popper, replace: boolean) {
     if(replace) {
       ADialog.removeAllPopups();
     }
     if(ADialog.visiblePopups.length === 0) {
       popup.ownerDocument.addEventListener('keyup', escKeyListener);
     }
+
+    const closePopupOnMouseLeave = () => {
+      if(ADialog.visiblePopups.indexOf(popup) < ADialog.visiblePopups.length - 1) {
```

Comment:
not the last one?

Reply:
`ADialog.visiblePopups[ADialog.visiblePopups.length - 1] !== popup`

Reply:
Yes, much clearer ;-)